magento/magento2
# Product images are being duplicated on import

### Description
Built on top of https://github.com/magento/magento2/pull/21146 and https://github.com/magento/magento2/pull/21855.

This PR adds image deletion so after import product will have images specified in the CSV. New images will be added, existing will not be duplicated and images not mentioned in the CSV will be removed. This PR has been created for 2.4-develop branch and when approved I'll create backport for 2.3 branch.

### Fixed Issues
magento/magento2#14398: Images Not Replaced via CSV Import
magento/magento2#21885: Product images are being duplicated on import

### Manual testing scenarios (*)

1. Import product CSV using Add/Update method multiple times
2. Check if image duplication happens or not
